src/sage/rings/infinity.py

```diff
@@ -481,7 +484,9 @@ def _div_(self, other):
 
     def __float__(self):
         r"""
-        Generate a floating-point infinity.  The printing of
+        Generate a floating-point infinity.
+
+        The printing of
         floating-point infinity varies across platforms.
```

Comment:
```suggestion
        The printing of floating-point infinity varies across platforms.
```
Looks slightly weird otherwise...